docs/src/content/guides/events.md

```diff
@@ -5,32 +5,25 @@ category: guides
 scope:
   - assign
 ---
-# Events
 
-Victory uses a flexible event system that is agnostic of event type. Browser events like `onClick` are handled identically to mobile touch events like `onPressIn`.  Victory's event system allows users to attach events to any rendered element, and trigger mutations on any other rendered element.
+# Events
```

Comment:
It looks like there were some prettier updates to the markup on save - I did verify that everything functions the same.